### PR TITLE
Change the webhook-configuration name

### LINKED ISSUE #4774: The name of the CRD's `webhook-configuration` is not precise
**What happened**:
The name of the CRD's webhook configuration is not precise。Now the name of the `webhook-configuraion` is `validate-devicemodel`. The first time I saw it, I thought there was something wrong with my configuration and only the `webhook-configuration` for devicemodel was running.
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

### PR DESCRIPTION
**Pull request by lishaokai1995**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #4774 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/admissioncontroller/admission.go [hunk 1]
```diff
@@ -158,9 +159,8 @@ func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, c
 	// validating webhook configuration
 	validatingWebhookConfiguration := admissionregistrationv1.ValidatingWebhookConfiguration{
 		ObjectMeta: metav1.ObjectMeta{
-			// TODO: this config name is not precise, think a way to change it more common, like `validate-kubeedge-crd`
-			// but there'll be two ValidatingWebhookConfigurations, to keep compatible, we can only keep one webhook for one CRD
-			Name: ValidateDeviceModelConfigName,
+			// there'll be two ValidatingWebhookConfigurations, to keep compatible, we can only keep one webhook for one CRD
+			Name: ValidateCRDWebhookConfigName,
```
Comment (fisherxu) [inline cloud/pkg/admissioncontroller/admission.go:163]: L30 can be deleted?

Comment (lishaokai1995) [reply cloud/pkg/admissioncontroller/admission.go:163]: > L30 can be deleted?
Yes, L30 can be deleted.

Comment (fisherxu) [reply cloud/pkg/admissioncontroller/admission.go:163]: Let's delete it, thanks.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**